project_resource_calendar/report/calendar_event_report.xml

```diff
@@ -20,8 +20,7 @@
             model="calendar.event"
             string="Calendar Event Report"
             report_type="qweb-pdf"
-            name="project_resource_calendar.event_report_view"
-            file="project_resource_calendar.event_report_view"
+            name="project_resource_calendar.calendar_event_report_view"
```

Comment:
@eilst  Please check there is no name calendar_event_report_view in the report folder. (Hope file name change already)